packages/api/Dockerfile

```diff
@@ -6,19 +6,26 @@ RUN npm install -g --force yarn
 FROM base as builder
 
 ENV NODE_ENV development
+
 ADD package.json .
-RUN yarn install --ignore-scripts
+RUN yarn install --ignore-scripts --frozen-lockfile
+
+ADD tsconfig.json .
 ADD src src
 RUN yarn run prepare
 
 FROM base
+
 ENV NODE_ENV production
+
 ARG DOCKER_TAG
 ENV DOCKER_TAG ${DOCKER_TAG}
 ARG GITHUB_SHA
 ENV GITHUB_SHA ${GITHUB_SHA}
+
 ADD package.json .
-RUN yarn install
+RUN yarn install --frozen-lockfile
```

Comment:
Hmm, I'm surprised this works absent the presence of a yarn.lock file.

Reply:
lol good catch. I'll add the yarn.lock here too, also expected it would have given me an error there.

Reply:
So I fixed this by running the `docker build` with the context of the repository root instead, so I could copy the `yarn.lock` from the root `package.json`. Felt a bit awkward to be honest, wish there was an easier way, but I think it works fine. Let me know if you have a better idea or rather keep it the way it was, without `yarn.lock` (seems risky tho). I don't really needed this change to get the `tsconfig.json` to work, so I can revert the other changes.